common/components/chrome/MainHeader.jsx

```diff
@@ -143,6 +143,18 @@ class MainHeader extends React.Component<{||}, State > {
     );
   }
 
+  _renderLogInLink(): void {
+    return (
+      <SectionLink
+          activeSection={this.state.activeSection}
+          key="LogIn"
+          section={Section.LogIn}
+          title="Log In"
+          target="_blank" rel="noopener noreferrer"
```

Comment:
This line doesn't do anything since it's not a proper link, and If it did, the Login page would open in a new tab which we don't want.